scripts/setup.js

```diff
@@ -8,14 +8,13 @@ import simpleGit from 'simple-git';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 
-const DATA_REPOSITORY_URL = `https://github.com/ambanum/CGUs-data.git`;
 const NEW_HISTORY_MESSAGE = 'Start a new local empty history';
-const DOWNLOAD_HISTORY_MESSAGE = `Download the entire history of terms of services from ${DATA_REPOSITORY_URL}`;
+const DOWNLOAD_HISTORY_MESSAGE = `Download the entire history of terms of services from ${config.get('history.publicSnapshotsRepository')} and ${config.get('history.publicVersionsRepository')}`;
 
 (async () => {
   try {
-    if (fsApi.existsSync(config.get('history.dataPath'))) {
-      return console.log(`It seems that the database is already initialized as the "${config.get('history.dataPath')}" directory already exists. Erase that folder first if you’d like to set up a new database.`);
+    if (fsApi.existsSync(config.get('history.versionsPath')) || fsApi.existsSync(config.get('history.snapshotsPath'))) {
+      return console.log(`It seems that the database is already initialized as the "${config.get('history.versionsPath')}" directory already exists. Erase that folder first if you’d like to set up a new database.`);
```

Comment:
This message does not reflect the conditions defined in the code.
```suggestion
      return console.log(`It seems that the database is already initialized as the "${config.get('history.versionsPath')}" or the "${config.get('history.snapshotsPath')}" directories already exist. Erase these folders first (or change the source path in the config) if you’d like to set up a new database.`);
```